source/server/getting-started/configuration/index.rst

```diff
@@ -2,81 +2,104 @@
-Dimension configuration files are used to affect a certain dimension or group of worlds. These types of configs will
-override the global config files. World configuration files are used to modify individual worlds only. World configs
-override dimension and global configs.
+After making your desired changes, save your file. If your server was running while you made changes you have to reload
+the config file using the reload commands explained :ref:`below <configuration_sponge_reload>`.
 
-Modifying the Config In-Game
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For example after making changes to your ``global.conf``, run ``/sponge reload -g`` to reload the global configurations
+from file.
 
-It is possible to modify these configs through the in-game command ``/sponge config``. The syntax for the config
+Overriding the Global Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Say you would like to override any of the global configs in just a single world or dimension type, for example raise
+the spawn cap of monsters in just the Vanilla Nether. Subconfigs allow you to do just that, inside the main Sponge
+config folder you can find a ``worlds`` folder. First sorted by mods, then dimension type, you can find configs for
+every world and dimension on your server. Simply add whatever you want to change to those configs and it will just
+apply in the related worlds. Overriding any less specific configs.
+This follows a simple order: World > Dimension > Global
+
+Modifying the Configuration Using Commands
+==========================================
+
+It is possible to modify most configs through the in-game command ``/sponge config``. The syntax for the config
 command looks like this:
 
 .. code-block:: none
 
     /sponge config <flag> <key> <value>
 
-There are *flags* for specifying the target that you would wish to change. These flags are global, dimension, and world.
+There are the following **flags** for specifying the target that you would wish to change.
 
-* ``-g`` is the flag for global
-* ``-d <dim>`` targets a dimension (replacing *<dim>* with the dimension you want to configure)
-* ``-w <world>`` targets one world (replacing *<world>* with your chosen world).
+* ``-g`` is the flag for global (``global.conf``)
+* ``-d <dim>`` targets a dimension (``dimension.conf``)
+* ``-w <world>`` targets one world (``world.conf``)
+* ``-t`` is the flag for tracker (``tracker.conf``)
 
-The *key* is the value you want to change. The *value* is whatever you want to change the value of the key to.
+``<name>`` being a placeholder, you need replace with the name of your target, e.g. ``-d minecraft:nether``.
 
-Here is an example of this command in action:
+The **key** is the value you want to change. The **value** is whatever you want to change the value of the key to.
+Say you have the following file ``config/sponge/minecraft/nether/dimension.conf`` and want to change `chunk-load`
```

Comment:
I'd prefer
"For example, if you want to change `chunk-load` to `true` in the file ``config/sponge/minecraft/nether/dimension.conf`` shown below:"

(I don't mind the addition, but is it really helpful to include a slice of the config here?)

Reply:
I really wanted to explain how the command format structure works, which each step separated by a "."